openedx/features/database_extract/migrations/0004_auto_20180425_0547.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('database_extract', '0003_targetcourse_email_list'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='targetcourse',
+            old_name='email_list',
+            new_name='emails',
+        ),
+    ]
```

Comment:
delete all existing migrations and create only one migration as all db related things are final now.

Reply:
done.